aten/src/ATen/native/TensorShape.cpp

```diff
@@ -909,7 +909,28 @@ Tensor view(const Tensor& self, IntArrayRef size) {
     "not compatible with input tensor's size and stride (at least one dimension"
     " spans across two contiguous subspaces). Use .reshape(...) instead.");
   auto stride_value = *stride;
-  auto self_ = self.clone();
+  Tensor self_;
+  if (self.is_quantized()) {
+    TORCH_CHECK(self.qscheme() == kPerTensorAffine,
+                "Only PerTensorAffine quantization is supported right now");
+    self_ = at::_empty_affine_quantized({0}, self.options(), self.q_scale(),
+                                        self.q_zero_point());
+  } else if (self.is_cuda()) {
+    // NOTE: This path of constructing the Tensor directly with the viewed
```

Comment:
where is this coming from?  shouldn't empty with the options below create a tensor on the correct device?

Also, not necessary for this PR, but we really need an "alias" that works across all tensor types.

Reply:
This comes from THCTensor_(view), should we remove it?

Reply:
you mean `at::alias`, there are a some legacy TH functions still used in aten, I think I can move them to aten if quantized Tensor needs them in the future.